idl/github.com/uber/cadence/shared.thrift

```diff
@@ -676,6 +676,7 @@ struct ChildWorkflowExecutionTerminatedEventAttributes {
 }
 
 struct HistoryEvent {
+  01:  optional i64 (js.type = "Long") version
```

Comment:
Can you add this after EventType as ID 35?

Reply:
sure